templates/zerver/api/delete-user.md

```diff
@@ -0,0 +1,34 @@
+# Delete a user
+
+Delete a user.
```

Comment:
This should probably be "Deactivate a user" and link to https://zulipchat.com/help/deactivate-or-reactivate-a-user.  And you should add the marker that this is an admin-only endpoint.